lib/Onyx.js

```diff
@@ -1376,7 +1397,6 @@ const Onyx = {
     multiSet,
     merge,
     mergeCollection,
-    hasPendingMergeForKey,
```

Comment:
Is this not used anywhere? Was this removed on purpose?

Reply:
Yes, this was removed on purpose. This is actually a private function and is not meant to be exported, yet i added it in the last [this PR](https://github.com/Expensify/react-native-onyx/pull/238), which was a mistake